app/views/versions/v9/death-arrears-payee/next-of-kin-payee.html

```diff
@@ -1,17 +1,9 @@
 {% from "date-input/macro.njk"    import govukDateInput %}
-{% from "fieldset/macro.njk"    import govukFieldset %}
-{% from "input/macro.njk"    import govukInput %}
 {% extends "versions/v1/bereavement-support-payments/layout.html" %}
 
 {% block formContent %}
-  {% set dapHtml %}
-    <div class="govuk-form-group dap-address">
-      {% set prefix = "dap" %}
-      {% include "includes/address-lookup.html" %}
-    </div>
-  {% endset %}
   <h1 class="govuk-heading-m">{{"death_arrears_who_we_will_pay" | translate}}</h1>
-  <form method="post" action="../payee-bank">
+  <form method="post" action="../check">
```

Comment:
`/notification-check` think `check` has the declaration and actually sends the form off

Reply:
Amended now 👍